Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:14.04
+WORKDIR /vagrant
+COPY . /vagrant
+
+RUN ./provision.sh
```

Comment:
Missing a trailing newline.

Also, `./provision.sh` uses `sudo` which doesn't really make sense to do within a `Dockerfile`.